ballerina/tests/test.bal

```diff
@@ -0,0 +1,188 @@
+// AUTO-GENERATED FILE. DO NOT MODIFY.
+// This file is auto-generated by the Ballerina OpenAPI tool.
+
+// Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/os;
+import ballerina/test;
+
+// Configurable variables for environment setup.
+configurable boolean isLiveServer = os:getEnv("isLiveServer") == "true";
+configurable string token = isLiveServer ? os:getEnv("OPENAI_API_KEY") : "test";
+configurable string serviceUrl = isLiveServer ? "https://api.openai.com/v1" : "http://localhost:9090";
+configurable string apiKey = isLiveServer ? token : "";
+
+// Initialize the connection configuration and client.
+final ConnectionConfig config = {auth: {token: apiKey}};
```

Comment:
Usually we break this into multiple lines when formatting. Shall we do the same in here?
```suggestion
final ConnectionConfig config = {auth: {token: apiKey}};
```